src/main/kotlin/racingcar/RacingGame.kt

```diff
@@ -0,0 +1,16 @@
+package racingcar
+
+class RacingGame(private val cars: Cars) {
+    private val positions: MutableList<Positions> = arrayListOf()
```

Comment:
클래스에 개념적으로 동일하지만 하나는 공개된 API의 일부이고 다른 하나는 구현 세부 사항인 두 개의 프로퍼티가 있는 경우 private 프로퍼티 이름의 접두사로 밑줄을 사용합니다. 이를 뒷받침하는 프로퍼티(backing property)라고 해요.

```suggestion
    private val _positions: MutableList<Positions> = arrayListOf()
    val positions: List<Positions>
        get() = _positions
```

Reply:
backing property 에 대해 개념이 없었는데 정확히 알게 되었어요. 🙇 
d86351a5944b21b37a5e5288de817692ef54460e 에서 적용하였습니다.